oct/ansible/oct/playbooks/package/ami-mark-ready.yml

```diff
@@ -55,4 +55,13 @@
         resource: '{{ origin_ci_aws_ami_id }}'
         tags:
           ready: 'yes'
-        state: present
\ No newline at end of file
+        state: present
+
+    - name: add any additional tags
```

Comment:
My mental picture of `oct package ami --mark-ready` was to treat it as a different command rather than an option to `oct package ami`. That's obviously not how it comes off, but is evident that if you use it you won't actually package it in that same run.